pkg/storage/create.go

```diff
@@ -0,0 +1,30 @@
+package storage
+
+import (
+	"fmt"
+	"os"
+)
+
+type Script struct {
+	Name    string
+	Content string
+}
+
+func Create(script *Script) error {
+	if err := CreateScriptDir(script.Name); err != nil {
+		return fmt.Errorf("unable to create script dir: %v", err)
+	}
+
+	scriptDir := GetScriptDir(script.Name)
+	indexFile, err := os.OpenFile(scriptDir+"/index.sh", os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0777)
+	if err != nil {
+		return fmt.Errorf("create: %v", err)
+	}
+	defer indexFile.Close()
+	if _, err = indexFile.WriteString(script.Content); err != nil {
+		return err
+	}
+	fmt.Println("Script created in: " + scriptDir)
+
+	return nil
+}
```

Comment:
Instead of making two files create.go and install.go I think we should merge these two files into script.go. So all the logic involving storing script files can be kept in this file. We can also include the logic to create the script dir in here too.

We might want storage to manage other things in the future.